Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionURLSchemeHandlerCocoa.mm

```diff
@@ -107,13 +109,15 @@
         }
 
         RefPtr<API::Error> error;
-        auto *fileData = extensionContext->extension().resourceDataForPath(requestURL.path().toString(), error);
-        if (!fileData) {
-            extensionContext->recordError(wrapper(error));
+        auto resourceData = extensionContext->extension().resourceDataForPath(requestURL.path().toString(), error);
+        if (resourceData.empty()) {
+            extensionContext->recordErrorIfNeeded(wrapper(error));
             task.didComplete([NSError errorWithDomain:NSURLErrorDomain code:NSURLErrorFileDoesNotExist userInfo:nil]);
             return;
         }
 
+        auto fileData = wrapper(API::Data::create(resourceData)).get();
```

Comment:
```suggestion
        auto *fileData = static_cast<NSData *>(API::Data::createWithoutCopying(resourceData)->wrapper());
```

Reply:
resourceData isn't an NSData, and createWithoutCopying only exists for std::span if we also pass in a freeDataFunction.